pom.xml

```diff
@@ -247,18 +247,51 @@ limitations under the License.]]></inlineHeader>
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/*$$accessor.class</exclude>
+            <exclude>**/*__MapperGenerated*.class</exclude>
```

Comment:
Do we need a `*` after `MapperGenerated`?

Reply:
@adutra yes, we need it to exclude inner classes

Reply:
@adutra Here is what you get otherwise

![image](https://user-images.githubusercontent.com/1618116/108106211-ed5e1e00-708d-11eb-8e85-9ed6da7e4eff.png)

Reply:
Huh. Ok then! 👍